editor/scene_tree_dock.cpp

```diff
@@ -2883,11 +2885,84 @@ void SceneTreeDock::_tree_rmb(const Vector2 &p_menu_pos) {
 
 void SceneTreeDock::_update_tree_menu() {
 	PopupMenu *tree_menu = button_tree_menu->get_popup();
-	tree_menu->set_item_checked(tree_menu->get_item_idx_from_text(TTR("Auto Expand to Selected")), EditorSettings::get_singleton()->get("docks/scene_tree/auto_expand_to_selected"));
+	tree_menu->clear();
```

Comment:
This also doesn't need to be cleared. Clearing is only useful when list of items can change.

Reply:
Right. This used to change before the warning message was turned into a tooltip. A lot of these initialisations on the bottom page are really messy, but it is what it is.

As I was writing this, however, I realised that by fixing this, icons do not load. Is it a good idea to not clear the filter menus in favour of writing more code around to ensure the menu items are set on NOTIFICATION_READY and the icons are always up to date?

Reply:
Eh, if it would require more code then maybe it's ok this way. It affects performance, but not in a meaningful amount.